packages/sanity/src/_internal/cli/actions/backup/downloadAsset.ts

```diff
@@ -0,0 +1,52 @@
+import path from 'path'
+import {createWriteStream} from 'fs'
```

Comment:
we are slowly moving to importing node imports using the new namespace way, to make it clear it's using node's internal import. Sorry it's not in the style guide.


```suggestion
import path from 'node:path'
import {createWriteStream} from 'node:fs'
```

Reply:
Added namespacing.